prow/plugins/plugins.go

```diff
@@ -1131,6 +1153,8 @@ func EventsForPlugin(name string) []string {
 	return events
 }
 
+// EnabledReposForPlugin returns the orgs and repos that have enabled the passed
```

Comment:
nit: The vestigial pythoner in me wants the first sentence short summary to stay on the first line.

I'd either figure out how to express the intent within 80 chars or just ignore the line length since golang doesn't seem to care about it

Reply:
Do we have a reasonably-agreed-upon max line length in this project? I'm super irritated by this consistency where sometimes it's 60 and others it's 120. 😗